### PR TITLE
[DQMGUI] updates on shift page plots of GUI of GEM offlineDQM

### PR DESCRIPTION
**Pull request by Dongwoon77**
**PR description:**

In this PR, update and extends the part of offlineDQM for GEM. The details are following:

- **efficiency vs eta using the global muons** 
- **residual distributions**

@jshlee @watson-ij @quark2

### CODE REVIEW

**Comment by nothingface0** [comment]
@Dongwoon77 could you add a new line at the end of the file to make it easier to patch? Thanks

File: dqmgui/layouts/shift_l1t_layout.py [hunk 1]
```diff
@@ -48,4 +48,9 @@ def l1tlayout(i, p, *rows): i["00 Shift/L1T/" + p] = DQMItem(layout=rows)
...
-
+l1tlayout(dqmitems,"23 - Ratio L1TMuons BX FirstBunch minus1",
+    [{'path': "L1T/L1TObjects/L1TMuon/timing/First_bunch/Ratio_L1TMuons_BX_FirstBunch_minus1", 'description': "Fraction of L1T Muon objects found in BX -1 over all L1T Muon objects found in a BX range of +/-2 BX around BX 0.", 'draw': { 'withref': "no" }}])
+l1tlayout(dqmitems,"24 - Ratio L1TJet BX FirstBunch minus1",
+    [{'path': "L1T/L1TObjects/L1TJet/timing/First_bunch/Ratio_L1TJet_BX_FirstBunch_minus1", 'description': "Fraction of L1T Jet objects found in BX -1 over all L1T Jet objects found in a BX range of +/-2 BX around BX 0.", 'draw': { 'withref': "no" }}])
+l1tlayout(dqmitems,"25 - Ratio L1TEGamma ptmin 20p0 gev BX FirstBunch minus1",
+    [{'path': "L1T/L1TObjects/L1TEGamma/timing/First_bunch/Ratio_L1TEGamma_ptmin_20p0_gev_BX_FirstBunch_minus1", 'description': "Fraction of L1T EGamma objects with pT >= 20 GeV found in BX -1 over all L1T EGamma objects with pT >= 20 GeV found in a BX range of +/-2 BX around BX 0", 'draw': { 'withref': "no" }}])
```
Comment (nothingface0) [inline dqmgui/layouts/shift_l1t_layout.py:56]:
```suggestion
    [{'path': "L1T/L1TObjects/L1TEGamma/timing/First_bunch/Ratio_L1TEGamma_ptmin_20p0_gev_BX_FirstBunch_minus1", 'description': "Fraction of L1T EGamma objects with pT >= 20 GeV found in BX -1 over all L1T EGamma objects with pT >= 20 GeV found in a BX range of +/-2 BX around BX 0", 'draw': { 'withref': "no" }}])

```

**Comment by nothingface0** [comment]
Actually, that's not the only problem, it seems that your branch has reverted some already existing commits, and does not include the changes you intended. Fix them and let us know.

**Comment by Dongwoon77** [comment]
Dear @nothingface0 

Thank you for your help.

I mistake the made of PR so I will make the new PR.

so can you close this PR?

Thank you,
Dongwoon,

**Comment by Dongwoon77** [comment]
@nothingface0 

Thank you for your help!

Dongwoon

**Pull request closed without merging**